plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoRestCatalog.java

```diff
@@ -613,7 +623,7 @@ public void updateViewColumnComment(ConnectorSession session, SchemaTableName sc
     private SessionCatalog.SessionContext convert(ConnectorSession session)
     {
         return switch (sessionType) {
-            case NONE -> new SessionContext(randomUUID().toString(), null, null, ImmutableMap.of(), session.getIdentity());
+            case NONE -> new SessionContext(randomUUID().toString(), null, getCredential(), ImmutableMap.of(), session.getIdentity());
```

Comment:
not sure if we should reuse token by passing `catalogName.toString()` instead of `randomUUID().toString()`